app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -138,6 +138,13 @@ def available_images
       templates
     end
 
+    def image_exists?(image)
+      client.templates.get(image)
+      true
+    rescue OvirtSDK4::NotFoundError
+      false
```

Comment:
It would be good to log exception using `Foreman::Logging.exception`.
What do you think?

Reply:
I agree I will add it.